cmd/sidecar/main.go

```diff
@@ -76,28 +98,29 @@ func main() {
 				return err
 			}
 
-			// ------------ START LOGIC ------------
-
-			k8sSecretsMap, err := secrets.RetrieveK8sSecrets()
-			if err != nil {
-				errLogger.Printf("Failed to retrieve k8s secrets: %s", err.Error())
-				return err
-			}
-
-			k8sSecretsMap, err = secrets.UpdateK8sSecretsMapWithConjurSecrets(k8sSecretsMap)
-			if err != nil {
-				errLogger.Printf("Failed to update K8s Secrets map: %s", err.Error())
-				return err
+			if conjurSecretsDest == 1 {
```

Comment:
i don't think we need this `if` clause. we already initialized `secretsHandler` according to the value of `conjurSecretsDest`. now we only need to run `secretsHandler.HandleSecrets`. am i wrong?

Reply:
just spoke about this with you and we decided to add a TODO at the top